tests/core/core.spec.ts

```diff
@@ -7,19 +7,27 @@ import { expect } from "chai";
 import { asPlainObject } from "df/tests/utils";
 import * as path from "path";
 
-const TEST_CONFIG: dataform.IProjectConfig = {
-  warehouse: "redshift",
-  defaultSchema: "schema"
-};
-
-const TEST_CONFIG_WITH_SUFFIX: dataform.IProjectConfig = {
-  ...TEST_CONFIG,
-  schemaSuffix: "suffix"
-};
+class TEST_CONFIG {
+  public static REDSHIFT: dataform.IProjectConfig = {
+    warehouse: "redshift",
+    defaultSchema: "schema"
+  };
+
+  public static REDSHIFT_WITH_SUFFIX: dataform.IProjectConfig = {
+    warehouse: "redshift",
+    defaultSchema: "schema",
+    schemaSuffix: "suffix"
+  };
```

Comment:
This is a less elegant method of duplicating the prior `REDSHIFT` that `  ...TEST_CONFIG,
  schemaSuffix: "suffix"`; this is because the variables have to be static in order to be used in the way they currently are.

Is there a workaround for this?

Reply:
Does it not work if you do: 
```
{
  ...TEST_CONFIG.REDSHIFT
}
```
?

Reply:
Yes that's it! cheers.